qual/src/main/java/com/virtuslab/qual/internal/NotUiThreadSafe.java

```diff
@@ -0,0 +1,3 @@
+package com.virtuslab.qual.internal;
+
+public @interface NotUiThreadSafe {}
```

Comment:
Pls add a reference to https://github.com/typetools/checker-framework/issues/3253